lcm.rake

```diff
@@ -74,35 +74,28 @@ end
 
 RSpec::Core::RakeTask.new(:test)
 namespace :test do
-  desc 'Run integration tests'
+  desc 'Run integration and slow tests'
   RSpec::Core::RakeTask.new(:integration) do |t|
-    t.pattern = 'spec/lcm/integration/**/*.rb'
+    t.pattern = 'spec/lcm/(integration|slow)/**/*.rb'
   end
 
   desc 'Run load tests'
   RSpec::Core::RakeTask.new(:load) do |t|
     t.pattern = 'spec/lcm/load/**/*.rb'
   end
 
-  desc 'Run slow tests'
-  RSpec::Core::RakeTask.new(:slow) do |t|
-    t.pattern = 'spec/lcm/slow/**/*.rb'
-  end
-
-  namespace :integration do
-    desc 'Run integration tests in Docker'
-    task :docker do
-      system('docker-compose -f docker-compose.lcm.yml up --force-recreate --abort-on-container-exit --exit-code-from appstore appstore') ||
+  namespace :docker do
+    desc 'Run integration and slow tests in Docker'
+    task :integration do
+      system('docker-compose -f docker-compose.lcm.yml up --force-recreate --abort-on-container-exit --exit-code-from appstore appstore bundle exec rake -f lcm.rake test:integration') ||
         fail('Test execution failed!')
     end
   end
-  namespace :load do
-    desc 'Run load tests in Docker'
-    task :docker do
-      system('docker-compose -f docker-compose.lcm.yml run --rm appstore bundle exec rake -f lcm.rake test:load')
 
-      system(check_exit_code) || fail('Test execution failed!')
-    end
+  desc 'Run load tests in Docker'
+  task :load do
+    system('docker-compose -f docker-compose.lcm.yml run --rm appstore bundle exec rake -f lcm.rake test:load') ||
+        fail('Test execution failed!')
```

Comment:
warning: Layout/MultilineOperationIndentation: Use 2 (not 4) spaces for indenting an expression spanning multiple lines. (rubocop)